docs/source/architecture.rst

```diff
@@ -0,0 +1,143 @@
+Drivers Atlas Testing Architecture
+==================================
+
+The Drivers Atlas Testing project validates the behavior of drivers against
+Atlas clusters. For example, a test could be to perform queries or writes
+in a loop while Atlas performs a maintenance operation, and assert that all
+queries and writes complete successfully.
+
+The project consists of the following principal components:
+
+- Astrolabe - the orchestration utility.
+- :doc:`Workload executors <spec-workload-executor>` - wrappers providing
+  a common interface to each driver's unified test runner.
```

Comment:
Can we link to the unified test format here when referring to it? I think it would be best to use a link to master.

Reply:
Done.